cdc/log.cc

```diff
@@ -543,6 +543,14 @@ static schema_ptr create_log_schema(const schema& s, std::optional<utils::UUID>
     auto add_columns = [&] (const schema::const_iterator_range_type& columns, bool is_data_col = false) {
         for (const auto& column : columns) {
             auto type = column.type;
+            if (type->get_kind() == abstract_type::kind::empty) {
+                auto& columns = s.all_columns();
```

Comment:
Please check, but I think we have s.regular_column_count() so you don't need to open-code this calculation here.

Reply:
done.